website/assets/js/pages/customers/new-license.page.js

```diff
@@ -0,0 +1,101 @@
+parasails.registerPage('new-license', {
+  //  ╦╔╗╔╦╔╦╗╦╔═╗╦    ╔═╗╔╦╗╔═╗╔╦╗╔═╗
+  //  ║║║║║ ║ ║╠═╣║    ╚═╗ ║ ╠═╣ ║ ║╣
+  //  ╩╝╚╝╩ ╩ ╩╩ ╩╩═╝  ╚═╝ ╩ ╩ ╩ ╩ ╚═╝
+  data: {
+    // Form data
+    formData: { /* … */ },
+
+    // For tracking client-side validation errors in our form.
+    // > Has property set to `true` for each invalid property in `formData`.
+    formErrors: { /* … */ },
+
+    quoteFormRules: {
+      numberOfHosts: {required: true},
+    },
+
+    billingFormRules: {
+      paymentSource: {required: true},
+    },
+
+    // Syncing / loading state
+    syncing: false,
+
+    // Server error state
+    cloudError: '',
+
+    // Success state when form has been submitted
+    cloudSuccess: false,
+    showBillingForm: false,
+    showQuotedPrice: false,
+    quotedPrice: undefined,
+    numberOfHostsQuoted: undefined,
+    thirtyDaysFromTodayInMS: Date.now() + 30*24*60*60*1000,
+    calendlyLink: '',
+    orderComplete: false,
+  },
+
+  //  ╦  ╦╔═╗╔═╗╔═╗╦ ╦╔═╗╦  ╔═╗
+  //  ║  ║╠╣ ║╣ ║  ╚╦╝║  ║  ║╣
+  //  ╩═╝╩╚  ╚═╝╚═╝ ╩ ╚═╝╩═╝╚═╝
+  beforeMount: function() {
+    let today = new Date(Date.now());
+    this.calendlyLink = 'https://calendly.com/fleetdm/demo?month='+today.getFullYear()+'-'+today.getMonth();
```

Comment:
nice work figuring this out!  But two things:

1. JavaScript's `getMonth()` function doesn't work like you might think it does.  For example, since it's January, `(new Date()).getMonth()` returns `0`.
2. you can actually just do `https://calendly.com/fleetdm/demo` and calendly will use the current date automatically